styles/helpers/tabs.css

```diff
@@ -0,0 +1 @@
+@mixin tabs 5, 6px;
```

Comment:
Left in the border-radius section of this because its quite possible that we'd want rounded corners and the logic that was used within the CSS is complex enough that I didn't want to lose it.